UPGRADING_2.2_TO_2.8.md

```diff
@@ -128,8 +128,11 @@ them from the old server instance to the new.
 
 The path is `/var/lib/ckan/default/storage/uploads/`.
 
-If the new server is accessible from outside, copying them directly should work, i.e.:
-```
-scp -r ubuntu@opendataphilly.org:/var/lib/ckan/default/storage/uploads \
-       NEW_USERNAME@NEW_SERVER:/var/lib/ckan/default/storage/
+Follow these instructions on the CKAN 2.8 instance:
+
+```bash
+$ scp -r ubuntu@OLD_SERVER_PRIVATE_IP:/var/lib/ckan/default/storage/uploads ./uploads
```

Comment:
Thanks for updating this part.  I'm realizing, though, that any kind of direct copy will only work if we haven't already stopped the old instance by the time the new one is provisioned.  Not sure if that means we should keep the old instance up until this step or make a local copy before stopping the old instance to upload to the new instance later.